### DISCUSSION TITLE
Add to quotes API  dealiasKeepOpaques

### DISCUSSION THREAD

**Discussion by rssh** [scala/scala3#18562] [Feature Requests]
Looks like opaque types are not represented in low-level quotes API,
 which means that macro can't determine - is expression has an opaque type, maybe non-opaque type aliased.

One possible solution is to add. dealiasKeepOpaques.

**Comment by nicolasstucki** [comment]
We should add this method.

**Reply by nicolasstucki** [reply]
Adding it in https://github.com/lampepfl/dotty/pull/18583
